### PR TITLE
add r-hillr

### PR DESCRIPTION
**Pull request by nick-youngblut**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jakirkham** [comment]
Can you please run?

```
git add --renormalize .
git commit -m "Renormalize recipe EOLs"
git push
```

**Comment by nick-youngblut** [comment]
> Can you please run?
> 
> ```
> git add --renormalize .
> git commit -m "Renormalize recipe EOLs"
> git push
> ```

@jakirkham, I tried running ` git add --renormalize .`, but it didn't do anything.

**Comment by jakirkham** [comment]
Ah ok. Thanks for giving that a try

**Comment by jakirkham** [comment]
@conda-forge/help-r, could someone please take a look at this PR?

File: recipes/r-hillr/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/hillR_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/hillR/hillR_{{ version }}.tar.gz
+  sha256: 36adecac41c0476af81d253bf5adf97e4f33cf634579a78c336322f1ff70e27b
+
+build:
+  merge_build_host: True  # [win]
+  skip: true  # [win]
```
Comment (cbrueffer) [inline recipes/r-hillr/meta.yaml:17]: Why is Windows skipped here?

Comment (cbrueffer) [reply recipes/r-hillr/meta.yaml:17]: If there's a good reason (e.g. dependency doesn't build on Windows etc) please add a comment above the skip with a short description.

**Pull request merged**